core/src/main/scala/matryoshka/package.scala

```diff
@@ -262,6 +262,12 @@ package object matryoshka extends CofreeInstances with FreeInstances {
     */
   def once[A](f: A => Option[A]): A => A = expr => f(expr).getOrElse(expr)
 
+  /** Converts a failable fold into a non-failable, by returning the default
+   * upon failure.
+    */
+  def onceDefault[A, B](f: A => Option[B], default: B): A => B =
+    expr => f(expr).getOrElse(default)
+
```

Comment:
Re naming, I like `orDefault` and `orOriginal`.

Re function parameters, the signature `(A => Option[B]) => (A => B)` is really the goal here (in both cases), so I think that `(default: B)(f: A => Option[B])` makes that signature more clear.